src/test/resources/random_csapatnev/virologist_field_interact.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Does Virologist interacts with Field properly?
+  Tests if a Virologist's Field interact does the proper actions.
+
+#  Scenario: Virologist interacts with Laboratory
```

Comment:
Szerintem kommentben is leirhatod, hogy foglalkoztal ezzel a scenarioval, csak nem tesztelhetonek bizonyult, es akkor torolhetoek a nyomai.

Reply:
Mivel a kódban bent hagytam kommentelve a kódot, így itt is itt hagytam a Scenario-t hozzá, így a későbbiekben nem kell kibogarászni a kódbol milyen Given statementek voltak, stb.